src/github.org

```diff
@@ -0,0 +1,99 @@
+#+TITLE: GitHub Training 2017
+#+AUTHOR: Sahit Chintalapudi, Josh Ting 
+#+EMAIL: schintalapudi@gatech.edu, josh.ting@gatech.edu
+#+OPTIONS: toc:nil timestamp:nil reveal_control:t num:nil reveal_history:t tags:nil author:nil
+
+* What is Git?
+- Git is a "Distributed Version Control System" (DCVS)
+- It is the tool RoboJackets uses to collaborate on code
+- Git tracks changes made to files in your project, and handles merging
+  these changes
+- Working on different parts of code and then combining them creates unique challenges that
+  cannot be handled by file sharing systems like DropBox and Google Drive
+
+* What is GitHub?
+- GitHub is a publically hosted git server
+- It hosts our projects as "repositories" on the internet
+- This saves us the hassle of hosting a server for fellow RoboJackets to
+  obtain copies of the code
+- As you matriculate through Tech, you'll be building an online portfolio of
+  your work on GitHub.
+
+* How do you find things on GitHub?
+- Repositories are stored at https://github.com/<ORG-OR-USER-NAME>/<REPO-NAME>
+- So all of our projects can be found at https://github.com/RoboJackets
+- Your personal projects will exist at https://github.com/<YOUR-GH-USERNAME>
+
+** Create a GitHub account
+- [[https://github.com/join%0A][https://github.com/join]]
+- Please include /at least/ your real first name
```

Comment:
I would discourage this from being a requirement, this will make some people uncomfortable.